pkg/visitors/find.go

```diff
@@ -113,10 +113,14 @@ func FindFilePredicate(r string) (func(f uefi.Firmware) bool, error) {
 	if err != nil {
 		return nil, err
 	}
+	insenRE, err := regexp.Compile("^(?i)" + r + "$")
```

Comment:
or ciRE for case insensitive regexp? not important.

Reply:
Done